api/Document.json

```diff
@@ -4210,6 +4210,92 @@
           }
         }
       },
+      "fullscreenchange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/fullscreenchange_event",
+          "description": "<code>fullscreenchanged</code> event"
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "64"
+              },
+              {
+                "version_added": "47",
+                "version_removed": "65",
```

Comment:
I'm thinking that these `version_removed: 65` statements should actually be 64.  Based upon your data, Firefox 64 was the version where it was no longer hidden behind a flag; thus, the requirement to use the flag was removed in Firefox 64.